<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenoirClass</class>
 <widget class="QMainWindow" name="RenoirClass">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>727</width>
    <height>514</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Renoir</string>
  </property>
  <property name="styleSheet">
   <string notr="true">sideBarLayout
{
	position: absolute;
	width: 20%;
	height: 100%;
	left: 80%;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>-40</y>
      <width>296</width>
      <height>505</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="sideBarLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="1" column="0">
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>R:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="redSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>G:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="greenSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>B:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="blueSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>255</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="1" column="2">
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>H:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="hueSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>359</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>S:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="saturationSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_5">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>V:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="valueSpin">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_6">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Pick color:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="color">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="0" column="2">
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Opacity:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="alphaSpin">
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>1.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Layers</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Normal</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Add</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lighten</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Multiply</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Screen</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Darken</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Color Burn</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Color Dodge</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Overlay</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Soft Light</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Hard Light</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Difference</string>
                 </property>
                </item>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="lockButton">
                <property name="text">
                 <string>Lock</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QListWidget" name="layers"/>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QPushButton" name="newButton">
                  <property name="text">
                   <string>New</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="removeButton">
                  <property name="text">
                   <string>Remove</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QPushButton" name="upButton">
                  <property name="text">
                   <string>Up</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="downButton">
                  <property name="text">
                   <string>Down</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QPushButton" name="renameButton">
                  <property name="text">
                   <string>Rename</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="hideButton">
                  <property name="text">
                   <string>Hide</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="canvasWidget" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>311</width>
      <height>391</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="mainMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>727</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuExport_as">
     <property name="title">
      <string>Export as...</string>
     </property>
     <addaction name="actionPortable_anymap"/>
     <addaction name="actionImage_sequence"/>
     <addaction name="actionBase64"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionInsert"/>
    <addaction name="separator"/>
    <addaction name="menuExport_as"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
    <addaction name="actionKernel"/>
    <addaction name="actionSharpen"/>
    <addaction name="actionGaussian_blur"/>
    <addaction name="actionEdge_detect"/>
    <addaction name="actionDither"/>
    <addaction name="actionTreshold"/>
   </widget>
   <widget class="QMenu" name="About">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuGenerate">
    <property name="title">
     <string>Generate</string>
    </property>
    <widget class="QMenu" name="menuFractal">
     <property name="title">
      <string>Fractal</string>
     </property>
     <addaction name="actionMandelbrot"/>
     <addaction name="actionJulia"/>
     <addaction name="actionLSystem"/>
    </widget>
    <addaction name="actionNoise"/>
    <addaction name="menuFractal"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuFilters"/>
   <addaction name="menuGenerate"/>
   <addaction name="About"/>
  </widget>
  <widget class="QToolBar" name="tools">
   <property name="allowedAreas">
    <set>Qt::LeftToolBarArea</set>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMove"/>
   <addaction name="actionBrush"/>
   <addaction name="actionEraser"/>
   <addaction name="actionRectangle"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionText"/>
   <addaction name="actionZoom_in"/>
   <addaction name="actionZoom_out"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="settingsBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="allowedAreas">
    <set>Qt::TopToolBarArea</set>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBrush_size"/>
   <addaction name="actionFill"/>
   <addaction name="actionShape_properties"/>
   <addaction name="actionText_properties"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionImage_sequence">
   <property name="text">
    <string>Image sequence</string>
   </property>
  </action>
  <action name="actionKernel">
   <property name="text">
    <string>Kernel</string>
   </property>
  </action>
  <action name="actionSharpen">
   <property name="text">
    <string>Sharpen</string>
   </property>
  </action>
  <action name="actionGaussian_blur">
   <property name="text">
    <string>Gaussian blur</string>
   </property>
  </action>
  <action name="actionEdge_detect">
   <property name="text">
    <string>Edge detect</string>
   </property>
  </action>
  <action name="actionDither">
   <property name="text">
    <string>Dither</string>
   </property>
  </action>
  <action name="actionTreshold">
   <property name="text">
    <string>Treshold</string>
   </property>
  </action>
  <action name="actionPortable_anymap">
   <property name="text">
    <string>Portable anymap</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionNoise">
   <property name="text">
    <string>Noise</string>
   </property>
  </action>
  <action name="actionMandelbrot">
   <property name="text">
    <string>Mandelbrot</string>
   </property>
  </action>
  <action name="actionJulia">
   <property name="text">
    <string>Julia</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionBrush">
   <property name="text">
    <string>Brush</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionMove">
   <property name="text">
    <string>Move</string>
   </property>
  </action>
  <action name="actionText">
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionBase64">
   <property name="text">
    <string>Base64</string>
   </property>
  </action>
  <action name="actionLSystem">
   <property name="text">
    <string>L-System</string>
   </property>
  </action>
  <action name="actionBrush_size">
   <property name="text">
    <string>Brush size</string>
   </property>
  </action>
  <action name="actionFill">
   <property name="text">
    <string>Fill</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionShape_properties">
   <property name="text">
    <string>Shape properties</string>
   </property>
  </action>
  <action name="actionText_properties">
   <property name="text">
    <string>Text properties</string>
   </property>
  </action>
  <action name="actionInsert">
   <property name="text">
    <string>Insert</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Renoir.qrc"/>
 </resources>
 <connections/>
</ui>
